device-types/Cisco/asr1002-hx.yaml

```diff
@@ -0,0 +1,46 @@
+manufacturer: Cisco
+model: ASR1002-HX
+slug: asr1002-hx
+u_height: 2
+is_full_depth: true
+console_ports:
+  - name: con 0
+power_ports:
+  - name: P0
+  - name: P1
+interfaces:
+  - name: GigabitEthernet0/0/0
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/1
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/2
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/3
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/4
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/5
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/6
+    type: 1000base-x-sfp
+  - name: GigabitEthernet0/0/7
+    type: 1000base-x-sfp
+  - name: Te0/0/0
```

Comment:
Needs to be the full form of the interface name (i.e. `TenGigabitEthernet`).

Reply:
Makes sense. I was going back and forth in my mind on these. A `show ip interface brief` would show it in the shortened fashion (not sure why). Versus the actual show interface. Will get that fixed up and checked on the other ones tonight.